src/api/events.js

```diff
@@ -41,18 +40,24 @@ function bindEvent (elem, event, handler) {
   elem.addEventListener(name, handler, capture);
 }
 
-function bindEvents (elem, events) {
+export default function (elem, events) {
+  var queue = [];
+  var ready = false;
+
   Object.keys(events).forEach(function (name) {
-    bindEvent(elem, name, events[name]);
+    var handler = events[name].bind(elem);
+    bindEvent(elem, name, function (e) {
+      if (ready) {
+        handler(e);
+      } else {
+        queue.push(handler.bind(elem, e));
```

Comment:
@nicolasv I ended up keeping the two `bind` calls here. The first one prevents the function from having to do `handler.call(elem, e)` within the wrapper handler every single time the event is handled. The second is so that the queue doesn't have to know the element and event objects to pass to the handlers. I removed one above in the `parseEvent` function that was unnecessary, though.

Happy to discuss changing the two binds above further if you still have any objections.